test/plugin/base_test.rb

```diff
@@ -182,7 +171,6 @@ def test_gce_used_when_detect_subservice_is_false
     setup_container_metadata_stubs
     d = create_driver(NO_DETECT_SUBSERVICE_CONFIG)
     d.run
-    assert_equal COMPUTE_CONSTANTS[:resource_type], d.instance.resource.type
```

Comment:
With the assertion removed, this test is not really doing much. We probably should get rid of it. However, we need to make sure every unit test / assertion we get rid of has some equivalent test at the entry level. (Previously we were checking the global variable. It's kind of like a shortcut.)